scripts/bootloader/provision.py

```diff
@@ -16,7 +16,9 @@
 def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output):
     # Add addresses
     provision_data = struct.pack('III', s0_address, s1_address, len(hashes))
-    provision_data += b''.join(hashes)
+    for mhash in hashes:
+        provision_data += struct.pack('I', 0xFFFFFFFF)
```

Comment:
I would add a comment here to say that this is some metadata used to invalidate the key

Reply:
Done